client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -267,12 +267,19 @@ const BallotAuditContest = ({
     }
   }
 
+  // Sort yes before no for propositions
+  const yesOption = contest.choices.find(c => c.name.toLowerCase() === 'yes')
+  const noOption = contest.choices.find(c => c.name.toLowerCase() === 'no')
```

Comment:
We can refine this logic if we encounter other situations that are clearly propositions but don't use yes or no as the options, e.g., affirm, reject, etc.